### DISCUSSION TITLE
Feature Request: Ability to toggle between multiple repos

### DISCUSSION THREAD

**Discussion by nickchomey** [gulbanana/gg#8] [Ideas]
It is currently very cumbersome to switch between repos in GG. We have to open the file explorer and select the repo path each time. 

It would be fantastic if when a repo is imported into/opened by GG, it gets added to some sort of repo shortlist that is accessible via a dropdown. 

GitButler does this and it makes it rapid to switch between repos that we are working on. 

![image](https://github.com/user-attachments/assets/8caf1f89-639f-44a2-b307-7bfd5399bbbc)

**Comment by tgecho** [comment]
I would love to add this if @gulbanana doesn't mind a PR. I think the least invasive change could be to replace the path in the bottom left of the window with a switcher that shows a list of recently opened paths.

We could also use this same list of paths to present a nicer initial "No Workspace Loaded" screen.

**Comment by gulbanana** [comment]
Sounds great. If you've got the time, I'd certainly accept it.

**Comment by gulbanana** [comment]
GG now supports multiple windows and a list of recently-opened repos in empty windows. On MacOS and Windows, the dock/taskbar icon also lets you launch into a repo or switch between open ones.
